graphql/ui_plugin_build_baron.go

```diff
@@ -8,17 +8,113 @@ import (
 	"time"
 
 	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/event"
```

Comment:
Note: I did not write this code. I moved it over from service with slight modifications so that it works.